static/app/views/traces/tracesChart.tsx

```diff
@@ -21,34 +22,50 @@ interface Props {}
 
 export function TracesChart({}: Props) {
   const location = useLocation();
-  const pageFilters = usePageFilters();
 
   const queries = useMemo(() => {
     return decodeList(location.query.query);
   }, [location.query.query]);
 
-  const spanIndexedCountSeries = useSpanIndexedSeries(
-    {
-      search: new MutableSearch(
-        queries
-          .filter(Boolean)
-          .map(q => `(${q})`)
-          .join(' OR ')
-      ),
-      yAxis: ['count()'],
-      interval: getInterval(pageFilters.selection.datetime, 'metrics'),
-      overriddenRoute: 'traces-stats',
-    },
-    'api.trace-explorer.stats'
-  );
+  const firstCountSeries = useTraceCountSeries(queries[0]);
+  const secondCountSeries = useTraceCountSeries(queries[1]);
+  const thirdCountSeries = useTraceCountSeries(queries[2]);
```

Comment:
As we discussed, we should be able to disable the 2nd and 3rd queries if they're unused